src/ensembl/src/shared/accordion/components/AccordionItem.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import DisplayName from '../helpers/DisplayName';
+import { DivAttributes } from '../helpers/types';
+import { nextUuid } from '../helpers/uuid';
+import { Provider as ItemProvider, UUID } from './ItemContext';
+import styles from '../css/Accordion.scss';
+
+type Props = DivAttributes & {
+  uuid?: UUID;
+};
+
+const defaultProps = {
+  className: styles.accordion__item
```

Comment:
Seems we are not using this style of class naming.

Reply:
That's now updated